dockerfiles/Picard/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM bashell/alpine-bash:latest
```

Comment:
Do you want to build on a fixed release (and put the release name in the tag)?  I'm not sure it's absolutely necessary, but floating tags makes it hard to guarantee you could build the image in the future.

Reply:
Yeah, I should have locked them down to a fixed version